config/rbac/kustomization.yaml

```diff
@@ -7,6 +7,7 @@ resources:
 - leader_election_role_binding.yaml
 - loadtest_editor_role.yaml
 - loadtest_viewer_role.yaml
+- pod_viewer_role.yaml
 - role.yaml
 - manager_binding.yaml
 - loadtest_component_bindings.yaml
```

Comment:
Since we are here, we might want to make the whole list alphabetically :)

Reply:
I will do that :)